src/pages/HomePage.tsx

```diff
@@ -21,23 +21,23 @@ const HomePage = (): JSX.Element => {
   };
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+    <div className="app">
+      <header className="header">
         <h1>Kvasir Movies</h1>
         {user !== null && <p>Welcome back, {user.email}!</p>}
-        <p>Find 🎬 with 👫 :D</p>
+        {user !== null && <MovieAdder user={user} />}
+        <p>Find 🎬 with 👫 😄</p>
         {user ? (
-          <button onClick={handleLogout}>Log Out</button>
+          <a onClick={handleLogout}>Log Out</a>
```

Comment:
Should Log Out also be in a links div?

Reply:
Oh it looks like maybe this was never cahnged, I updated it though